src/main/java/com/sap/sgs/phosphor/fosstars/model/value/DoubleValue.java

```diff
@@ -14,7 +14,7 @@
   /**
    * A double value.
    */
-  private final double number;
```

Comment:
I am not sure if the access level should be updated. At least, it should be package-private. Or, others can just use the `value()` method.